ktor-server/ktor-server-test-host/jvmAndNix/src/io/ktor/server/testing/TestApplication.kt

```diff
@@ -193,7 +211,24 @@ public class ApplicationTestBuilder : TestApplicationBuilder(), ClientProvider {
 }
 
 /**
- * Creates a test using [TestApplication]
+ * Creates a test using [TestApplication].
+ * To test a server Ktor application, do the following:
+ * 1. Use [testApplication] function to set up a configured instance of a test application running locally.
+ * 2. Use the [HttpClient] instance inside a test application to make a request to your server,
+ * receive a response, and make assertions.
+ *
```

Comment:
I think that it's worth mentioning that this builder will apply config from `application.conf` file if it exists